packages/react/src/components/Footer/LegalNav.js

```diff
@@ -76,6 +75,24 @@ function renderListItems(links) {
       </li>
     );
   });
+
+  renderedLinks.push(renderTrusteItem());
+
+  const chunked_arr = [];
+  let index = 0;
+
+  while (index < renderedLinks.length) {
+    chunked_arr.push(renderedLinks.slice(index, 2 + index));
```

Comment:
@RaphaelAmadeu just thinking about this iteration, if there's for some reason more than 6 links does this then give an unwanted column? I'm wondering if we should instead of have a hardcoded `2`, we should take the length of the `renderedLinks`, and do something like `Math.ceil(renderedLinks.length / 3)`. That way, it would be always 3 columns, but can support an infinite number of legal links. Not saying we'd ever reach that point, but the legal links can also be customized with custom navigation data.

Reply:
@jeffchew , I've applied this logic on the latest PR thank you!